test/pytest/test_data/streaming/stream_handler.py

```diff
@@ -0,0 +1,237 @@
+import json
+import logging
+import types
+from copy import copy
+
+import torch
+from transformers import AutoTokenizer, AutoModelForCausalLM
+from ts.torch_handler.base_handler import BaseHandler
+
+logger = logging.getLogger(__name__)
+
+
+class StreamingHandler(BaseHandler):
```

Comment:
Should we move this handler to ts_handler/distrubuted or move the core function to handler_utils/distributed?